src/payment/strategies/adyenv2/adyenv2-payment-strategy.ts

```diff
@@ -289,9 +300,23 @@ export default class AdyenV2PaymentStrategy implements PaymentStrategy {
             const state = {
                 ...newState.data.paymentMethod,
                 origin: window.location.origin,
+                browserInfo: this._getBrowserInfo(),
             };
 
             this._stateContainer = JSON.stringify(state);
         }
     }
+
+    private _getBrowserInfo(): BrowserInfoRequest {
+        return {
+            userAgent: navigator.userAgent,
+            acceptHeader: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,/;q=0.8',
+            language: navigator.language,
+            colorDepth: screen.colorDepth,
+            screenHeight: screen.height,
+            screenWidth: screen.width,
+            timeZoneOffset: new Date().getTimezoneOffset(),
```

Comment:
According to [Adyen's docs](https://docs.adyen.com/checkout/3d-secure/native-3ds2/api-reference#browserinfo) this should be a string, but `getTimezoneOffset` returns a number.